lib/v8.js

```diff
@@ -226,6 +226,11 @@ function getHeapCodeStatistics() {
   };
 }
 
+function setHeapSnapshotNearHeapLimit(limit) {
+  validateNumber(limit, 'limit', 1);
```

Comment:
I think we can simply return or throw an error if `getOptionValue('--heapsnapshot-near-heap-limit') > 0` here. Similarly the check for calling this method more than once can be done here, instead of in C++.